app/js/controllers/foyer/ressources/ressources.js

```diff
@@ -34,7 +47,11 @@ angular.module('ddsApp').controller('FoyerRessourcesCtrl', function($scope, $sta
         var types = _.chain(ressources).pluck('type').unique().filter(function(type) {
             return !_.contains(['pensionsAlimentairesVersees'], type);
         });
+
         types.forEach(function(type) {
+            if (type === 'caAutoEntrepreneur') {
```

Comment:
Either keep the current heavyweight yoda-style + triple-equal, or switch to a simpler double-equal, var-first style.